src/components/pastAnalysis/PastAnalysisTable.tsx

```diff
@@ -25,9 +24,17 @@ interface tableRowsChild {
   cells: { [title: string]: ReactNode }[]
 }
 
+interface TableStates {
+  page: number,
+  maxFeedId: number | undefined,
+  lastOffset: number,
+  lastPage: number,
+  storedPages: StudyInstanceWithSeries[][]
+}
+
 const PastAnalysisTable = () => {
   const { state: {
-    prevAnalyses: { page, perpage, totalResults, areNewImgsAvailable, listOfAnalysis },
```

Comment:
just to clarify here: we no longer use the `page` property from `prevAnalyses` since you now created a state object for it in this component?

Reply:
Yes, that is correct. The current page number is never used by any other component in the app, so there wasn't a need to be keeping in the store.